modules/lvms-creating-lvmcluster-using-rhacm.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-creating-lvmcluster-using-rhacm_{context}"]
+= Creating an LVMCluster CR by using {rh-rhacm}
+
+After you have installed {lvms} by using {rh-rhacm}, you must create an `LVMCluster` custom resource (CR).
+
+.Prerequisites
+
+* You have installed {lvms} by using {rh-rhacm}.
+* You have access to the {rh-rhacm} cluster using an account with `cluster-admin` permissions.
+* You read the "About the LVMCluster custom resource" section. See the "Additional resources" section.
+
+.Procedure
+
+. Log in to the {rh-rhacm} CLI using your {product-title} credentials.
+
+. Create a `ConfigurationPolicy` CR YAML file with the configuration to create an `LVMCluster` CR:
++
+.Example `ConfigurationPolicy` CR YAML file to create an `LVMCluster` CR
+[source, yaml]
+----
+apiVersion: policy.open-cluster-management.io/v1
+kind: ConfigurationPolicy
+metadata:
+  name: lvms
+spec:
+  object-templates:
+  - complianceType: musthave
+    objectDefinition:
+      apiVersion: lvm.topolvm.io/v1alpha1
+      kind: LVMCluster
+      metadata:
+        name: my-lvmcluster
+        namespace: openshift-storage
+      spec:
+        storage:
+          deviceClasses: <1>
+# ...
+            deviceSelector: <2>
+# ...
+            thinPoolConfig: <3>
+# ...
+            nodeSelector: <4>
+# ...
+  remediationAction: enforce
+  severity: low
+----
+<1> Contains the configuration to assign the local storage devices to the LVM volume groups.
+<2> Contains the configuration to specify the paths to the devices that you want to add to the LVM volume group, and force wipe the devices that are added to the LVM volume group. 
+<3> Contains the LVM thin pool configuration.
+<4> Contains the configuration to choose the nodes on which you want to create the LVM volume groups. If this field is empty, then all nodes without no-schedule taints are considered.
+
+. Create the `ConfigurationPolicy` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <file_name> -n <ocp-cluster-namespace> <1>
```

Comment:
On second thought, should it be explicitly saying `ocp-` here? This assembly AFAICT does not appear in the OSD or ROSA docs, but unless I'm missing a conditional related to this module in particular, the assembly does appear in the OKD docs [here](https://docs.okd.io/latest/storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.html). And so it would look strange for the nearby `{product-title}`s to be resolving to "OKD", with this hard-coded `ocp-` in the replaceable:

<img width="585" alt="image" src="https://github.com/openshift/openshift-docs/assets/3442316/4e32f106-8b10-4609-8ee6-5b7b22981494">

Reply:
@adellape 
It makes sense to update it to "<cluster_namespace".